src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
```

Comment:
Consider using the `gap` property instead of `margin-right` to add space between the stars. This would make the code cleaner and easier to maintain. This is a code quality issue.

Reply:
in instruction to this task recommended to avoid gap, because it will block npm tasks